src/pages/registration/new_registration/EditRegistration.tsx

```diff
@@ -42,15 +43,17 @@ const EditRegistration: FC = () => {
     setExpanded(isExpanded ? panel : false);
 
   return !showForm ? (
-    <>
+    <BackgroundDiv>
```

Comment:
Brukes BackgroundDiv her bare for å fikse bredden? 
tenker du i så fall å endre det når bredden på app blir fiksa?
Hvis det bare er for bredden er det vel like greit og bare bruke plain `<div>` også

Reply:
ja, har brukt BackgroundDiv for å få lik padding på alle sider. Men kan forsåvidt lage en ny komponent som ikke inneholder theme-funksjonalitet men kun padding 👍